app/experimenter/visualization/api/v3/models.py

```diff
@@ -67,6 +67,9 @@ def append_population_percentages(self):
         total_population = 0
         branches = {}
         for jetstream_data_point in self.__root__:
+            if jetstream_data_point.segment != Segment.ALL:
+                continue
```

Comment:
I love the way this is written now.  Logic clearly laid out, properties clear, constants clear, it's so much nicer to review now @emtwo 🎉 🎉 🎉 🎉 🎉